.github/scripts/close_nonexistent_disable_issues.py

```diff
@@ -3,54 +3,67 @@
 import multiprocessing as mp
 import os
 import re
+import sys
 import tempfile
-from typing import Any, Dict, List, Optional, Tuple
+from pathlib import Path
+from typing import Any, Dict, List, Tuple
 
 import requests
-import rockset  # type: ignore[import]
 from gitutils import retries_decorator
 
 
+REPO_ROOT = Path(__file__).resolve().parent.parent.parent
+sys.path.insert(0, str(REPO_ROOT))
+from tools.testing.clickhouse import query_clickhouse
+
+
+sys.path.pop(0)
+
+
 LOGS_QUERY = """
 with
     shas as (
         SELECT
-            push.head_commit.id as sha,
+            distinct
+            push.head_commit.id as sha
         FROM
-            commons.push
+            -- Not bothering with final here
```

Comment:
I learn from @ZainRizvi that the use (or abuse) of having final on the query could be the reason why some of my queries run OOM.  I guess I need to go back and double check some of them